ecosystem/gstreamer_plugin/README.md

```diff
@@ -146,6 +146,34 @@ These are also general parameters accepted by plugins, but the functionality the
 | 48 kHz        | `48000` |
 | 96 kHz        | `96000` |
 
+#### 2.3.3 PTS controlled pacing
+
+For selected plugins
+- SMPTE ST 2110-20 transmission plugin mtl_st20p_tx
+- SMPTE ST 2110-30 transmission plugin mtl_st30p_tx
+- SMPTE ST 2110-40 transmission plugin mtl_st40p_tx
+
+There is a feature called user-controlled pacing that allows you to control the
+pacing of packet transmission using the Presentation timestamp (PTS) of the buffer.
+As long as the PTS fits within the pacing window, packets will be sent at the exact
+time specified by the PTS.
+
+This feature requires:
+- Your system clock to be synchronized via PTP.
+- GStreamer buffers to provide a 64-bit timestamp (TUI) since epoch (PTP time).
+- MTL to be synchronized via ptp (onboard or user-controlled).
```

Comment:
How as a user of this plugin should I synchronize MTL? It is not clear. Should we add some flags to select between onboard or user-controlled. if not, please add info that MTL will synchronize itself to system clock by default.

Reply:
yea fixed